url_shortener_proj/database/migrations/2020_04_29_183613_create_urls_table.php

```diff
@@ -13,13 +13,12 @@ class CreateUsersTable extends Migration
      */
     public function up()
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('urls', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
-            $table->rememberToken();
+            $table->text('original');
```

Comment:
How many characters can fit in the text field?

Reply:
65,535 bytes, you think it's too much? Should I limit it?

Reply:
Nope, it's ok, you can leave it as is. VARCHAR(2000) would be the most optimal solution.